main.cpp

```diff
@@ -0,0 +1,58 @@
+//CodeU: Q1
+//
+//  Created by Julia Semavina on 17.05.2017
+//
+//  problem specification:
+//      Given two strings, write a method to decide if one is a permutation of the other
+//
+//  comments:
+//      register is ignored
+//
+
+#include <iostream>
+#include <map>
+
+
+char lowercase(char a) {
+    if (a <= 'Z' && a >= 'A')
+        return (a - ('Z' - 'z'));
+    return a;
+}
+
+bool is_permutation(std::string &A, std::string &B) {
```

Comment:
Since you are not modifying the strings is better to make that explicit:
bool is_permutation(const std::string &A, const std::string &B) {